frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -197,7 +197,7 @@ export class PieceEditor implements OnInit {
     }
     this.currentSelectedPiece = piece;
     this.urlPointingUUID = piece ? piece.uuid : '';
-    this.currentType = this.getCurrentTypeCode(piece.type);
+    this.currentType = piece ? this.getCurrentTypeCode(piece.type) : 0;
```

Comment:
Does `0` have some meaning? Can we create a named constant to represent it to avoid the "magic number"?

Reply:
It is the offset into a type name array for display purposes. 0 is an empty string.

Reply:
I changed the name of the currentType field to to currentTypeOffset